makeReducedConfig.py

```diff
@@ -287,7 +311,7 @@ def isIncluded(self, n, mem):
 parser.add_argument("-w", "--wires", type=str, default="wires.dat", help="Reference wires.dat file (from full config)")
 parser.add_argument("-p", "--process", type=str, default="processingmodules.dat", help="Reference processingmodules.dat file (from full config)")
 parser.add_argument("-m", "--memories", type=str, default="memorymodules.dat", help="Reference memorymodules.dat file (from full config)")
-parser.add_argument("-s", "--sector", type=str, default="F", help="TC/TP phi sector from which to create the reduced config")
+parser.add_argument("-s", "--sector", type=str, default="F", help="TC/TP phi sector from which to create the reduced config, e.g. "C" or "All" for all regions")
```

Comment:
@aryd The CI tests are complaining about this line, because of the double quotes in the help message, I think:
```suggestion
parser.add_argument("-s", "--sector", type=str, default="F", help="TC/TP phi sector from which to create the reduced config, e.g. \"C\" or \"All\" for all regions")
```

Reply:
I have fixed this now (I used single quotes...)